### ISSUE TITLE
SX1101 incorrectly reported for local variable that hides field

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#1706]
SX1101 is reported for the use of `this.` in the following code. The code fix results in a compilation error.

```csharp
class ClassName
{
  int items;
  void Method()
  {
    var items = this.items;
  }
}
```

**Comment by vweijsters** [comment]
Grabbing this
